src/main/java/net/minecraftforge/event/entity/living/EnderManAngerEvent.java

```diff
@@ -0,0 +1,26 @@
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.world.entity.monster.EnderMan;
+import net.minecraft.world.entity.player.Player;
+import net.minecraftforge.eventbus.api.Cancelable;
+
+/**
+ * Event that is fired when an enderman checks if the player is looking at him.
+ * If cancelled the enderman does not get angry at the player.
+ */
+@Cancelable
```

Comment:
This could benefit from `@HasResult`:
- `DENY`: do not let the enderman get angry (regardless of what `isEnderMask` says)
- `ALLOW`: allow the enderman to get angry (regardless of what `isEnderMask` says)
- `DEFAULT`: refer to vanilla behavior (i.e. call `ItemStack#isEnderMask`)